tools/export/exporters.py

```diff
@@ -168,7 +172,10 @@ def scan_and_copy_resources(self, prj_paths, trg_path, relative=False):
 
         # And add the configuration macros to the toolchain
         self.config_macros = config.get_config_data_macros()
-                
+
+        # Add the configuration file to the target directory
+        self.config_header =  "mbed_conf.h"
+        cfg.get_config_data_header(join(trg_path, self.config_header))
```

Comment:
cfg is not defined in this context ``NameError: global name 'cfg' is not defined``